spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -147,16 +127,11 @@
 # disable all repos not provided by SUSE Manager.
 DISABLE_LOCAL_REPOS=1
 
-# Disable yasts automatic online update feature in case it is enabled
-# on the client. Leaving automatic online update enabled, the client would
-# continue to update himself independently from SUSE Manager requests.
-DISABLE_YAST_AUTOMATIC_ONLINE_UPDATE=1
-
 # SUSE Manager Specific settings:
 #
-# - Alternate location of the client tool repos providing the zypp-plugin-spacewalk
-# and packges required for registration. Unless they are already installed on the
-# client this repo is expected to provide them for SLE-10/SLE-11 based clients:
+# - Alternate location of the client tool repos providing 
+#   packges required for registration. Unless they are already installed on the
```

Comment:
typo, not introduced here but as you update the comment it's a good time to fix it.
```suggestion
#   packages required for registration. Unless they are already installed on the
```

Reply:
fixed